superset/assets/src/utils/restActions.js

```diff
@@ -0,0 +1,24 @@
+export function createPostPayload(payloadformat, data) {
+  let payload = JSON.parse(JSON.stringify(payloadformat)); // deepclone hack
+  replacePlaceholder(payload, data)
+  return payload;
+}
+
+const replacePlaceholder = (obj, data) => {
+  Object.keys(obj).forEach(key => {
+    let value = obj[key]
+    if (isString(value)) {
```

Comment:
It would be great if you can use `_.isString` from lodash.

Reply:
done